package.json

```diff
@@ -2,6 +2,7 @@
   "name": "news-explorer-frontend",
   "version": "0.1.0",
   "private": true,
+  "homepage": "http://localhost:3000",
```

Comment:
* You need to deploy the app
* In ` README.md` you need to add some info about your project: technologies, how to download, how to start, how to deploy the app. And you need to add the link to your **deployed app**.